src/spaceone/dashboard/model/domain_dashboard_model.py

```diff
@@ -59,6 +59,10 @@ class DomainDashboard(MongoModel):
             'viewers',
             'labels',
             'user_id',
-            'domain_id'
+            'domain_id',
+            {
```

Comment:
Setting unique_with as a field property automatically creates a unique index.
So please remove the current index code.